docs/buildah-run.md

```diff
@@ -14,6 +14,12 @@ the *buildah config* command.
 
 ## OPTIONS
 
+**--tty**
```

Comment:
... standard output is attached to a terminal.  Though we might want to change it to point to check standard input instead.